### PR TITLE
Remove centos 6 support

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: kitchen.yml [hunk 1]
```diff
@@ -58,10 +57,9 @@ suites:
     driver:
       name: terraform
       variables:
-        centos_image: "CentOS 8.1"
+        centos_image: "CentOS 8.2"
```
Comment (ramereth) [inline kitchen.yml:60]:
```suggestion
        centos_image: "CentOS 8.3"
```

File: variables.tf [hunk 2]
```diff
@@ -8,7 +8,7 @@ variable "centos_image" {
     default = "CentOS 7.7"
 }
 variable "ssh_key_name" {
-    default = "bootstrap"
+    default = "kennedy"
```
Comment (ramereth) [inline variables.tf:11]: Revert this please.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 2 above)
Comment (TheREK3R) [reply variables.tf:11]: Is there a bootstrap key I should be using for this? I have just been changing it to my key to use, but I often forget to exclude that change as above.

Comment (ramereth) [reply variables.tf]: Yes, it's on fir and fair to copy it and use it.

**Review by ramereth — APPROVED** [review]

**Comment by ramereth** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v4.3.0.  Have a nice day!

**Pull request merged**